ccm_web/client/src/models/canvas.ts

```diff
@@ -140,3 +140,8 @@ export const injectCourseName = (sections: CanvasCourseSection[], courseName: st
 export function sortSections<T extends CanvasCourseSection> (sections: T[]): T[] {
   return sections.sort((a, b) => { return a.name.localeCompare(b.name, undefined, { numeric: true }) })
 }
+
+export interface CanvasUserCondensed {
+  loginId: string
```

Comment:
I think this still needs to be updated to be `login_id`? I guess it didn't come up as an issue cause you aren't using the property, but we should probably fix it.